Source/Core/VideoCommon/CommandProcessor.cpp

```diff
@@ -215,7 +217,13 @@ void CommandProcessorManager::RegisterMMIO(MMIO::Mapping* mmio, u32 base)
                    system_.GetFifo().RunGpu();
                  }));
 
-  mmio->Register(base | PERF_SELECT, MMIO::InvalidRead<u16>(), MMIO::Nop<u16>());
+  // TODO
+  mmio->Register(base | PERF_SELECT, MMIO::DirectRead<u16>(&m_perf_select),
+                 MMIO::DirectWrite<u16>(&m_perf_select, 0x0007));
+
+  // TODO
```

Comment:
What's the TODO?

Reply:
I've not figured out what the register does, only that it exists because it preserves one byte of data.

Reply:
I think there should be a little explanation like that at least for both TODOs.

Reply:
Added some notes.